src/main/java/com/tvtracker/enterprise/dao/MediaEntryDAOStub.java

```diff
@@ -52,7 +53,7 @@ public MediaEntry save(MediaEntry mediaEntry) throws Exception {
      * @return MediaEntry object representation of corresponding database record
      */
     @Override
-    public MediaEntry fetch(int entryId) throws Exception {
```

Comment:
This is a stub.  How do you know that the implementation won't throw an exception?  That is likely.

This was covered in a previous code review as well.